Failing tests against master

```
======================================================================
FAIL: test_add_user_to_group_remove_user_from_group (__main__.TestUserAndGroupActions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "owncloud/test/test.py", line 889, in test_add_user_to_group_remove_user_from_group
    self.assertEquals(e.exception.status_code, 102)
AssertionError: 101 != 102

----------------------------------------------------------------------
Ran 53 tests in 123.776s
```

Could also be a regression on OC master.

--- a/apps/provisioning_api/lib/users.php
+++ b/apps/provisioning_api/lib/users.php
@@ -388,9 +388,14 @@ class Users {
 			return new OC_OCS_Result(null, \OCP\API::RESPOND_UNAUTHORISED);
 		}
 
-		$group = $this->groupManager->get(!empty($parameters['_delete']['groupid']) ? $parameters['_delete']['groupid'] : null);
+		$group = !empty($parameters['_delete']['groupid']) ? $parameters['_delete']['groupid'] : null;
 		if($group === null) {
 			return new OC_OCS_Result(null, 101);
+		}
+
+		$group = $this->groupManager->get($group);
+		if($group === null) {
+			return new OC_OCS_Result(null, 102);
 		}
 
 		$targetUser = $this->userManager->get($parameters['userid']);

--- a/apps/provisioning_api/tests/userstest.php
+++ b/apps/provisioning_api/tests/userstest.php
@@ -1357,6 +1357,16 @@ class UsersTest extends OriginalTest {
 		$this->assertEquals($expected, $this->api->removeFromGroup(['userid' => 'TargetUser', '_delete' => ['groupid' => 'TargetGroup']]));
 	}
 
+	public function testRemoveFromGroupWithNoTargetGroup() {
+		$loggedInUser = $this->getMock('\OCP\IUser');
+		$this->userSession
+			->expects($this->once())
+			->method('getUser')
+			->will($this->returnValue($loggedInUser));
+		$expected = new \OC_OCS_Result(null, 101);
+		$this->assertEquals($expected, $this->api->removeFromGroup(['userid' => 'TargetUser', '_delete' => []]));
+	}
+
 	public function testRemoveFromGroupWithNotExistingTargetGroup() {
 		$loggedInUser = $this->getMock('\OCP\IUser');
 		$this->userSession
@@ -1369,7 +1379,7 @@ class UsersTest extends OriginalTest {
 			->with('TargetGroup')
 			->will($this->returnValue(null));
 
-		$expected = new \OC_OCS_Result(null, 101);
+		$expected = new \OC_OCS_Result(null, 102);
 		$this->assertEquals($expected, $this->api->removeFromGroup(['userid' => 'TargetUser', '_delete' => ['groupid' => 'TargetGroup']]));
 	}
 

